node/src/components/consensus/era_supervisor.rs

```diff
@@ -64,7 +64,7 @@ use crate::{
 ///
 /// A node keeps `2 * BONDED_ERAS` past eras around, because the oldest bonded era could still
 /// receive blocks that refer to `BONDED_ERAS` before that.
-const BONDED_ERAS: u64 = DEFAULT_UNBONDING_DELAY - AUCTION_DELAY;
+const BONDED_ERAS: u64 = DEFAULT_UNBONDING_DELAY - 3;
```

Comment:
This cannot be hardcoded. It has to be the same as `AUCTiON_DELAY` in the EE.

Reply:
Yeah, that was the one part I am kinda stuck on, the story itself isn't done, this was a draft, and I am still working on it

Reply:
Well, the PR isn't marked as Draft (there's an option to open a Draft rather than fully-fledged PR), you've assigned people to review and you even got an approval. There was nothing preventing you from merging it (and breaking consensus).

Reply:
Oh yeah, sorry about that

Reply:
No worries, it's easy to miss things and stuff happens so quickly that we can't keep an eye on everything all the time.

Reply:
Marked as a draft now

Reply:
@goral09 are these changes sufficient